core/plannernotes.c

```diff
@@ -418,14 +418,19 @@ void add_plan_to_notes(struct diveplan *diveplan, struct dive *dive, bool show_d
 					mingas_depth = get_depth_units(lastbottomdp->depth.mm, NULL, &depth_unit);
 					/* Print it to results */
 					if (cyl->start.mbar > lastbottomdp->minimum_gas.mbar) snprintf(mingas, sizeof(mingas),
```

Comment:
Is this really if and snprintf on the the same line? Could you please split this in two lines (I know, you did not touch this recently but it would be a good place to do so).

Reply:
Ahhh,..
Lubomir also tried to teach me but I still seem to be blind when it's about bad coding style.
Did an update.